server/src/controllers/AuthController.js

```
@@ -0,0 +1,83 @@
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
+const uuidv4 = require("uuid").v4;
+const users = require("../data/users.json");
+const Validation = require("../utilities/Validation");
+const { addUsers } = require("../helpers");
+
+const parsedUsers = JSON.parse(JSON.stringify(users));
+
+const signup = async (req, res) => {
+  const validationRes = Validation.validateSignup(req.body);
+  if (validationRes.status) {
```

can use fail fast approach here